src/services/TrackerService.ts

```diff
@@ -93,9 +93,13 @@ export async function getTrackersForFlaws(requestBody: TrackersFilePost) {
 export function trackerUrl(type: string, id: string): string {
   switch (type) {
     case 'BUGZILLA':
-      return (new URL(`/${id}`, osimRuntime.value.backends.bugzilla)).href;
+      return osimRuntime.value.backends.bugzilla
+        ? (new URL(`/${id}`, osimRuntime.value.backends.bugzilla)).href
+        : 'http://bugzilla-service:8001';
```

Comment:
Maybe an _or_ `||` inside the URL function looks better?
```suggestion
      return (new URL(`/${id}`, osimRuntime.value.backends.bugzilla || 'http://bugzilla-service:8001')).href;
```

Reply:
Totally +1